src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureCause.java

```diff
@@ -663,6 +641,61 @@ public String getDisplayName() {
             return "";
         }
 
+        /**
+         * Form validation for {@link #description}. Checks for not empty and not "Description..."
+         *
+         * @param value the form value.
+         * @return {@link hudson.util.FormValidation#ok()} if everything is well.
+         */
+        @RequirePOST
+        public FormValidation doCheckDescription(@QueryParameter final String value) {
+            if (Util.fixEmpty(value) == null) {
```

Comment:
you should probably check `Item.Configure` here

Reply:
Why? The only check done here is whether the description is the default description.

Reply:
Not `Item.CONFIGUE` but `com.sonyericsson.jenkins.plugins.bfa.PluginImpl#VIEW_PERMISSION` or rather `com.sonyericsson.jenkins.plugins.bfa.PluginImpl#UPDATE_PERMISSION`.

Before when the check methods where on the object instance they where protected behind those permissions and not routable to if the user didn't have it. Now when they are moved to the descriptor they are routable through `jenkins.model.Jenkins#getDescriptorByName`.
The description check is not that important, but `doCheckName` has a potential of leaking information.

Reply:
Added